src/Dialect/ONNX/Transforms/Decompose.cpp

```diff
@@ -942,7 +944,7 @@ class ReplaceCastLikeByCastPattern : public OpRewritePattern<ONNXCastLikeOp> {
 
     // Replace
     Value res = onnx_mlir::OnnxBuilder(rewriter, loc)
-                    .cast(input, saturate, TypeAttr::get(outputType));
+                    .cast(input, output, saturate, TypeAttr::get(outputType));
```

Comment:
@imaihal Is there a reason why you want to pass in the `output` here ?

Reply:
Done. Thanks.